stdlib/public/Glibc/module.freebsd.map.in

```diff
@@ -0,0 +1,364 @@
+//===--- module.map -------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// This is a semi-complete modulemap that maps glibc's headers in a roughly
+/// similar way to the Darwin SDK modulemap. We do not take care to list every
+/// single header which may be included by a particular submodule, so there can
+/// still be issues if imported into the same context as one in which someone
+/// included those headers directly.
+///
+/// It's not named just FreeBSD libc so that it doesn't conflict in the event of a
+/// future official FreeBSD libc modulemap.
+module SwiftGlibc [system] {
```

Comment:
`SwiftGlibc`?

I almost feel like this file should be generated with gyb for both platforms from the same source...  Gyb supports substitutions for `@GLIBC_INCLUDE_PATH@`.

Do you think this would make sense?